__tests__/app.test.js

```diff
@@ -0,0 +1,50 @@
+const request = require('supertest');
+const db = require('../db/connection');
+const app = require('../app');
+const seed = require('../db/seeds/seed');
+const { articleData, commentData, topicData, userData} = require('../db/data/test-data/index');
+
+afterAll(() => {
+  if (db.end) db.end();
+});
+  
+beforeEach(() => seed({articleData, commentData, topicData, userData}));
+
+describe("1) 404 : invalid path", () => {
+it("Responds with 404 if client makes a request to an invalid path", () => {
+return request(app)
+.get('/api/topisc')
+.expect(404)
+.then(({ body : { message }}) => {
+  expect(message).toBe('Invalid path');
```

Comment:
when we send error messages back on our own error throw, we are better off using 'msg' as the key instead of 'message' so that we can differentiate our messages (custom errors) from things being thrown by the system (PSQL etc.)